features/administrators/adding_new_administrator_account_with_administration_role.feature

```diff
@@ -0,0 +1,16 @@
+@managing_administration_roles
+Feature: Adding a new Administrator account with administration role
+    In order to add new Administrator account to the system
```

Comment:
`In order to manage permissions of new Administrators`

Reply:
**a**dministrators?